### ISSUE TITLE
Add to maven Central Repository

### ISSUE THREAD

**Issue by ChrisMCMine** [kritts/HandWave#1]
It would be cool for all gradle/maven users if the library was published to the maven central repository.

**Comment by jairocgr** [comment]
Yes.. would be nice. Right now, i'm struggling trying to make HandWave works on Android Studio.

**Comment by tajchert** [comment]
Any update on that? It is useless otherwise (not being able to run it in Android Studio).

**Comment by kritts** [comment]
This is currently in the works. Hope to have an update soon!

**Comment by greenspand** [comment]
Is the library still in active development. Would it be available fro Android Studio aswell ?

**Comment by kritts** [comment]
The library is not currently in active development. Manishearth has created a branch that appears to work on Android Studio: 
https://github.com/kritts/HandWave/issues/3
